generators/app/index.js

```diff
@@ -27,7 +27,7 @@ module.exports = yeoman.generators.Base.extend({
   },
 
   install: function () {
-    if (this.options['skip-install']) {
+    if (this.options['skip-install'] || this.options.skipInstall) {
       return;
     }
 
```

Comment:
We can create it if it doesn't exist maybe? `--save` inside a generator really looks like a code smell